docs/docs/data-structures/tree/schema-evolution/allowed-types-rollout.mdx

```diff
@@ -94,6 +89,8 @@ const configC = new TreeViewConfiguration({
     schema: schemaC,
 });
 const viewC = treeC.viewWith(configC);
+assert.equal(viewC.compatibility.canView, false);
+assert.equal(viewC.compatibility.canUpgrade, true);
```

Comment:
Added these asserts to document what the client code should expect the compatibility values to be.